updater/reports/ReportOrgHistory.py

```diff
@@ -0,0 +1,55 @@
+from .ReportDaily import *
+
+# Lists the total number of organizations
+class ReportOrgHistory(ReportDaily):
+	def name(self):
+		return "org-history"
+
+	def updateDailyData(self):
+		newHeader, newData = self.parseData(
+			self.executeQuery(self.query())
+		)
+		self.header = newHeader if newHeader else self.header
+		self.data.extend(newData)
+		self.truncateData(self.timeRangeTotal())
+		self.sortDataByDate()
+
+	# Collects the number of organizations
+	def subquery(self, userType):
+		query = '''
+			SELECT
+				COUNT(*) AS count
+			FROM
+				users
+			WHERE
+				users.type = "Organization"
+			AND
+				users.login <> "github-enterprise"
```

Comment:
Good catch! We need to ignore this invalid organization in other reports, too. I think it would be easiest to hard code "github-enterprise" to https://github.com/Autodesk/hubble/blob/master/updater/reports/Report.py#L198-L210

//cc @pluehne ?